ocean_lib/models/data_nft_factory.py

```diff
@@ -88,11 +88,13 @@ def deploy_erc721_contract(
                 name,
                 symbol,
                 template_index,
-                additional_metadata_updater,
-                additional_datatoken_deployer,
+                ERC721TokenFactoryBase.to_checksum_address(additional_metadata_updater),
```

Comment:
is this not a child of ContractBase? can we use the class itself or ContractBase instead of an intermediary parent?